README.md

```diff
@@ -0,0 +1,159 @@
+**Supermarket**
+------------------
+
+
+
+
+About the Project
+------------------
+
+The following project titled as **Supermarket** is a web application that is designed to manage the repository of the Supermarket.
+Users can add, delete or view existing products in the repository using the application. Every product as record has certain associated characterestics with it that helps users identify the product. Every product has a unique product code and a unique ID along with a product name and price.
+***
+
+Using the Application
+----------------------
+
+In the following section, you will get to know how to use the application. Once you have cloned the repository, go to *$GOPATH/src/folder-name-you-cloned-it-to/* in your Goland terminal and type the following: 
+
+`go run ./*.go`
```

Comment:
I did not have the time, but I'd like you to figure out why this is a thing that needs to be done. I'm legit curious.
Also, try building a binary and running a binary instead. That use case should also work